roles/stressng/tasks/main.yaml

```diff
@@ -66,22 +77,38 @@
   when: resource_state.resources[0].status.state == "ConfigMaps Created"
 
 - block:
-  - name: wait for benchmark job to finish
-    k8s_facts:
-      kind: Job
-      api_version: batch/v1
-      name: '{{ meta.name }}-workload-{{ trunc_uuid }}'
-      namespace: "{{ operator_namespace }}"
-    register: stressng_workload_pod
+  - block:
+    - name: wait for benchmark job to finish
+      k8s_facts:
+        kind: Job
+        api_version: batch/v1
+        name: '{{ meta.name }}-workload-{{ trunc_uuid }}'
+        namespace: "{{ operator_namespace }}"
+      register: stressng_workload_pod
 
-  - operator_sdk.util.k8s_status:
-      api_version: ripsaw.cloudbulldozer.io/v1alpha
-      kind: Benchmark
-      name: "{{ meta.name }}"
-      namespace: "{{ operator_namespace }}"
-      status:
-        state: Benchmark Complete
-        complete: true
-      when: stressng_workload_pod | json_query('resources[].status.succeeded')"
+    - operator_sdk.util.k8s_status:
+        api_version: ripsaw.cloudbulldozer.io/v1alpha
+        kind: Benchmark
+        name: "{{ meta.name }}"
+        namespace: "{{ operator_namespace }}"
+        status:
+          state: Benchmark Complete
+          complete: true
+        when: stressng_workload_pod | json_query('resources[].status.succeeded')"
+    when: resource_kind == "pod"
 
+  - block:
+      - name: get complete
+        command: "redis-cli get complete"
+        register: complete_status
```

Comment:
Add a new line "\n"

Reply:
Fix it in the following commit:
https://github.com/cloud-bulldozer/benchmark-operator/pull/522/commits/c2b61762ecca7eb5b47ed598536bde610420a124